qml.qrc

```diff
@@ -162,5 +162,9 @@
         <file>pages/Keys.qml</file>
         <file>images/menu.png</file>
         <file>images/appicon.ico</file>
+        <file>fonts/DejaVuSans-Bold.ttf</file>
+        <file>fonts/DejaVuSans-BoldOblique.ttf</file>
+        <file>fonts/DejaVuSans-Oblique.ttf</file>
+        <file>fonts/DejaVuSans.ttf</file>
```

Comment:
I see some fonts already embedded in qml.qrc:

```
        <file>fonts/SFUIDisplay-Medium.otf</file>
        <file>fonts/SFUIDisplay-Regular.otf</file>
        <file>fonts/SFUIDisplay-Light.otf</file>
        <file>fonts/SFUIDisplay-Bold.otf</file>
```
Could those files be repurposed?

Reply:
Oh, those were added recently in #1206. Closing